lib/src/command_runner.dart

```diff
@@ -107,6 +109,11 @@ class VeryGoodCommandRunner extends CommandRunner<int> {
 
   @override
   Future<int?> runCommand(ArgResults topLevelResults) async {
+    if (topLevelResults.command?.name == 'completion') {
```

Comment:
what about `install-completion-files`?

Reply:
For these, we don't have that mich restriction on time. Completion has to be fast.